src/useq/_mda_sequence.py

```diff
@@ -143,6 +161,10 @@ def validate_positions(cls, v: Any) -> Any:
                 return list(v)
         return v
 
+    @validator("tile_plan", pre=True)
+    def validate_tileplan(cls, v: Any) -> Union[dict, NoTile]:
```

Comment:
this doesn't look necessary.  you don't _have_ to have a validator for each field (only if you want to do something special).  Here, you're just return the default if `v` is Falsey... but that's exactly what the `default_factory` you specified above already does.   this can go

Reply:
actually I think this is necessary in `validate_mda` that is validating the full MDASequence, am I wrong?

```python
@root_validator
def validate_mda(cls, values: Dict[str, Any]) -> Dict[str, Any]:
    if "axis_order" in values:
        values["axis_order"] = cls._check_order(
            values["axis_order"],
            z_plan=values.get("z_plan"),
            stage_positions=values.get("stage_positions", ()),
            channels=values.get("channels", ()),
            tile_plan=values.get("tile_plan"),
        )
    return values
```